.github/workflows/ci.yaml

```
@@ -24,13 +24,14 @@ jobs:
             matrix:
                 php: [ 8.1, 8.2, 8.3 ]
                 laravel: [ 10.*, 11.* ]
```

@danharrin I think that should be 
```suggestion
                laravel: [ ^10.0, ^11.0 ]
```
to match the `include` and `include` statements.